Wazi_Deploy/Reporting/List-packages-by-timewindow-query/Gitlab/README.md

```diff
@@ -0,0 +1,65 @@
+# IBM Wazi Deploy query and reporting pipeline template
+
+This template provides a [.gitlab-ci.yml](.gitlab-ci.yml) definition file to set up a pipeline that lists deployed packages within a specified time window by querying deployment information from Wazi Deploy evidence files.
+
+## Overview and capabilities
+This pipeline queries Wazi Deploy evidence files to extract and list details of packages deployed during a specific time window, generating a report of deployed packages within the specified timeframe.
+
+The pipeline leverages a custom query template to filter deployment information based on the time range provided.
+
+The pipeline consists of a single stage:
+
+`Query`
+   * Index the evidence files to enable searchability
```

Comment:
Is the indexing done in this pipeline? I thought it was done in the build/release pipelines